connect/src/main/java/com/stripe/android/connect/webview/StripeConnectWebView.kt

```diff
@@ -424,15 +445,41 @@ internal class StripeConnectWebView private constructor(
         )
     }
 
-    private fun WebView.evaluateSdkJs(function: String, payload: JsonObject) {
+    private fun WebView.evaluateSdkJs(
+        function: String,
+        payload: JsonObject,
+        resultCallback: ValueCallback<Result<String>>? = null
+    ) {
         val command = "$ANDROID_JS_INTERFACE.$function($payload)"
+        // language=JavaScript
+        val wrappedCommand = """
+            (function () {
```

Comment:
👀 Add support for error handling. Without this, the result is `"null"` when there is a JS error.